site/docs/tutorials/snake/moving-the-snake.md

```diff
@@ -0,0 +1,202 @@
+---
+sidebar_label: Moving the snake
```

Comment:
Is `sidebar_label` needed? I think docusaurus should be able to infer it from the first title in the document?

Reply:
I tried? I think. I'll check again and if it's not necessary, I'll remove it. I've seen your docs without them, so I was confused, why this didn't work for me.

But like I said: I'll check again.